### DISCUSSION TITLE
Layers in a GeoReferenced PDF

### DISCUSSION THREAD

**Discussion by toepferjoshua** [OpenOrienteering/mapper#2039] [Questions & Answers]
I have a Geo Referenced PDF from the USGS.  The PDF has both the satellite view and the topo view built in as layers.  When I open it in the normal PDF viewer it shows the Topo lines.  When I open it in the OpenOrienteering Mapper it shows the satellite view and I want the topo view.  How can I get it to show the other layer in the pdf?

**Comment by dg0yt** [comment]
I lack a similar file for testing. There are limitations in the PDF support in Mapper. It is based on GDAL and Poppler. GDAL brings a configuration option to select particular layers, but using global config can only be a workaround.

**Reply by toepferjoshua** [reply]
The file in particular is at the following address.  https://ngmdb.usgs.gov/topoview/viewer/#15/37.7338/-97.2834
The 2018 version and the PDF is the one.  One thing I noticed also was the KMZ download is also not correctly geo referenced (it doesn't line up with the pdf).
